src/value/owned.rs

```diff
@@ -53,6 +49,7 @@ impl ImportSource for OwnedImportSource {
 }
 
 /// An owned implementation of [`SymbolToken`].
+/// Here text as `None` represents SID $0
```

Comment:
This comment belongs on the trait definition (since it should be uniformly implemented that way).